src/Compilers/Core/Portable/Diagnostic/SourceLocation.cs

```diff
@@ -38,12 +38,13 @@ public SourceLocation(in SyntaxToken token)
         public SourceLocation(in SyntaxNodeOrToken nodeOrToken)
             : this(nodeOrToken.SyntaxTree!, nodeOrToken.Span)
         {
-            RoslynDebug.Assert(nodeOrToken.SyntaxTree is object);
+            Debug.Assert(nodeOrToken.SyntaxTree is object);
         }
 
         public SourceLocation(in SyntaxTrivia trivia)
-            : this(trivia.SyntaxTree, trivia.Span)
+            : this(trivia.SyntaxTree!, trivia.Span)
         {
+            RoslynDebug.Assert(trivia.SyntaxTree is object);
```

Comment:
I'm confused as to why one of these usages uses Debug and the other uses RoslynDebug.

Reply:
I started this PR before moving us to `netcoreapp3.1`. Thought I killed all the `RoslynDebug` uses but looks like I missed one.